lib/components/artist/shows.js

```diff
@@ -0,0 +1,56 @@
+/* @flow */
+'use strict';
+
+import React from 'react-native';
+const { View, Text, StyleSheet, Image } = React;
+
+import Headline from '../text/headline';
+import SerifText from '../text/serif'
+
+export default class Shows extends React.Component {
+	
+	render() {
+	  return (
+	  	<View style={styles.container}>
+		  	<SerifText style={styles.title}> Current & Upcoming Shows </SerifText>
+				<Image style={styles.thumbnail} />
+				<Text style={styles.sansSerifText}> {this.props.show.partner_name.toUpperCase()} </Text>
+				<Text style={styles.sansSerifText}> {this.showTypeString(this.props.show)} </Text>
+				<SerifText style={{marginTop: 2}}> {this.props.show.title} </SerifText>
+				<SerifText style={{color: 'grey', marginTop: 2}}> {this.dateAndLocationString(this.props.show)} </SerifText>
+			</View>
+		);
+	}
+	
+	showTypeString(show) {
+		return show.type.toUpperCase() + ', ' + show.number_of_works + ' WORKS'
```

Comment:
http://stackoverflow.com/a/29972553/95397

I assume that means that because you already had real tabs in this file, it auto-detected that it should keep using tabs.